src/adaptix/_internal/morphing/concrete_provider.py

```diff
@@ -75,6 +75,51 @@ def datetime_format_dumper(data: datetime):
         return datetime_format_dumper
 
 
+@dataclass
+@for_predicate(datetime)
+class DatetimeTimestampProvider(LoaderProvider, DumperProvider):
+    tz: timezone = timezone.utc
```

Comment:
Wouldn't be it better, if timezone would be received in `__init__`?

Reply:
it's dataclass what means less boilerplate and it's consistent with DatetimeFormatProvider that also receive format parameter through dataclass

Reply:
Oh didn't notice that it is daraclass, my bad